domain-model/src/test/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculatorTest.java

```diff
@@ -109,7 +109,61 @@ public void totalTillTodayShouldReturnCalculatedTotalAmountPlusInterestWhenCalcu
                     .build()
             ).build();
 
-        assertThat(TotalAmountCalculator.totalTillDateOfIssue(claimNoInterest))
+        assertThat(TotalAmountCalculator.totalTillDateOfIssue(claimStandardInterest))
+            .isEqualTo(Optional.of(format(new BigDecimal("60"))));
+    }
+
+    @Test
+    public void totalTillTodayWhenClaimIssuedOnIsToday() {
+        Claim claimStandardInterest = SampleClaim.builder()
+            .withClaimData(
+                SampleClaimData.builder()
+                    .withAmount(SampleAmountBreakdown.validDefaults())
+                    .withFeeAmount(TWENTY_POUNDS_IN_PENNIES)
+                    .withInterest(SampleInterest.standard())
+                    .withInterestDate(SampleInterestDate.submission())
+                    .build()
+            )
+            .withIssuedOn(LocalDate.now())
+            .build();
+
+        assertThat(TotalAmountCalculator.totalTillToday(claimStandardInterest))
+            .isEqualTo(Optional.of(format(new BigDecimal("60"))));
+    }
+
+    @Test
+    public void totalTillTodayWhenClaimIssuedOnIsTomorrow() {
+        Claim claimStandardInterest = SampleClaim.builder()
+            .withClaimData(
+                SampleClaimData.builder()
+                    .withAmount(SampleAmountBreakdown.validDefaults())
+                    .withFeeAmount(TWENTY_POUNDS_IN_PENNIES)
+                    .withInterest(SampleInterest.standard())
+                    .withInterestDate(SampleInterestDate.submission())
+                    .build()
+            )
+            .withIssuedOn(LocalDate.now().plusDays(1))
+            .build();
+
+        assertThat(TotalAmountCalculator.totalTillToday(claimStandardInterest))
+            .isEqualTo(Optional.of(format(new BigDecimal("60"))));
+    }
+
+    @Test
+    public void totalTillTodayWhenClaimIssuedOnIsDayAfterTomorrow() {
```

Comment:
I think we should replace this with the scenario where case was issued yesterday. In that case I would expect total amount to interest for one day.

Reply:
added yesterday's scenario as well.

Reply:
I would be happy to remove that scenario as this tests the same execution path as previous test.